src/Components/Components/src/ComponentBase.cs

```diff
@@ -124,7 +124,8 @@ protected virtual bool ShouldRender()
         => true;
 
     /// <summary>
-    /// Method invoked after each time the component has been rendered.
+    /// Method invoked after each time the component has been rendered interactively. This method is not
+    /// invoked during prerendering or server-side rendering.
```

Comment:
```suggestion
    /// Method invoked after each time the component has rendered interactively and the UI has finished
    /// updating (for example, after elements have been added to the browser DOM). You can use this method
    /// to take any needed actions on the resulting UI elements.
    ///
    /// This method is not invoked during prerendering or server-side rendering, because those processes
    /// are not attached to any live browser DOM and are already complete before the DOM is updated.
```

... in case we want to clarify *why* this is not invoked in prerendering/SSR.

Reply:
Nice! This looks like helpful content. Should this still go in the summary? What's the guidance for API docs on how much should go in the summary vs the remarks?

Reply:
This is indeed much more clear, thanks!